src/components/Articles.jsx

```diff
@@ -1,23 +1,39 @@
 import { getArticles } from "../utils/api";
+import { capitaliseStr } from "../utils/capitaliseStr";
 import { useState, useEffect } from "react";
 import { ArticleCard } from "./ArticleCard";
 import '../styles/Articles.css'
-import { Link, useParams } from "react-router-dom";
+import { Link, useParams, useSearchParams, useNavigate } from "react-router-dom";
 import { CircularProgress } from "@mui/material";
+import InputLabel from '@mui/material/InputLabel';
+import MenuItem from '@mui/material/MenuItem';
+import FormControl from '@mui/material/FormControl';
+import Select from '@mui/material/Select';
 
 export function Articles() {
 
     const [articles, setArticles] = useState([])
     const [isLoading, setIsLoading] = useState(true)
+    const [sortbyInput, setSortByInput] = useState('created_at')
+    const [orderInput, setOrderInput] = useState('desc')
+
+    const [searchParams, setSearchParams] = useSearchParams()
+
+    const sortOption = searchParams.get("sort_by") || "created_at"
+    const orderOption = searchParams.get("order") || "desc"
+
```

Comment:
I think you will need to set these as your sortbyinput and orderinput states in order to change the url and fetch new articles.